### ISSUE TITLE
21: Last kill cam

### ISSUE THREAD

**Issue by theonlypwner** [acreloaded/acr#39]
**Last kill cam** by `ShadowXDify`
Issue 21 posted to Google Code on 2012 Jul 6 at 22:10:41

Current status: Acknowledged
Current labels: `Type-Enhancement`, `Priority-Trivial`, `Component-Logic`, `Component-Engine`, `Component-Net`, `Effect-Protocol`, `Effect-Gameplay`

`ShadowXDify` on 2012 Jul 6 at 22:10:41:
> This is just a long term idea that would probably take a while but last kill cams would be epic for this game, I know that a friend of mine would also like this ( {ak}brandon ) and it would be an epic edition to the game. 

> Me and my friend like to do fancy stuff and sometimes trick shots ( quick scopes, 360 no scopes, etc. and it would add a whole ne element to the game that would alow us to see who made the best trick shot or all around shot.

> Idea: I have done some game making my self so if one of you has a copy of  on of the COD games then maybe look at the code and try to match it to assault cube.

`theonlypwner` on 2012 Jul 6 at 23:48:17:
> It might be hard since demo recording is server-sided, versus a client-side demo recording for replay. Or maybe the server can "replay" by sending old packets to clients...

> **Labels**: Component-Logic Component-Engine Component-Net Gameplay Protocol
> **Status**: Acknowledged

`ShadowXDify` on 2012 Jul 7 at 20:58:29:
> Sorry I mean that you could just display the last kill not the best one, that would be hard.

`theonlypwner` on 2012 Jul 9 at 00:23:56:
> This is blocked by Issue 28.

> **Labels**: -Priority-Medium Priority-Trivial
> **Blocked by**: 28

`theonlypwner` on 2012 Oct 18 at 22:03:57:
> Issue 62 has been merged into this issue.

`harryspag10` on 2012 Oct 19 at 07:45:28:
> What does any of ^^ that mean?

`theonlypwner` on 2014 Jan 16 at 01:11:54:
> At least we have a Battlefield-like "killcam" now.

**Comment by theonlypwner** [comment]
Issue 28 = #46
Issue 68 = ?
